src/main/java/net/tiagofar78/prisonescape/bukkit/BukkitMenu.java

```diff
@@ -0,0 +1,93 @@
+package net.tiagofar78.prisonescape.bukkit;
+
+import java.util.List;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import net.tiagofar78.prisonescape.game.PrisonEscapeItem;
+import net.tiagofar78.prisonescape.managers.MessageLanguageManager;
+
+public class BukkitMenu {
```

Comment:
Why menu?

Reply:
I think it is a good name. It is a box that opens in front of you to represent something. In this case it was the vault menu where the player could store and take items from the vault using that menu.